streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1649,18 +1765,33 @@ private void testThrowingDurringCommitTransactionException(final RuntimeExceptio
         assertEquals(1, thread.readOnlyActiveTasks().size());
     }
 
-    @Test
-    public void shouldNotCloseTaskAndRemoveFromTaskManagerIfProducerGotFencedInCommitTransactionWhenSuspendingTasks() throws Exception {
-        testThrowingDurringCommitTransactionException(new ProducerFencedException("Producer is fenced"));
-    }
-    @Test
-    public void shouldNotCloseTaskAndRemoveFromTaskManagerIfInvalidPidMappingOccurredInCommitTransactionWhenSuspendingTasks() throws Exception {
-        testThrowingDurringCommitTransactionException(new InvalidPidMappingException("PidMapping is invalid"));
-    }
-
-    @Test
-    public void shouldReinitializeRevivedTasksInAnyState() throws Exception {
-        final StreamsConfig config = new StreamsConfig(configProps(false));
+    @ParameterizedTest
+    @CsvSource({
+        "false, false",
```

Comment:
Could you use `methodSource` to offer override configs instead of those true-false?